cfg/CFG.cc

```diff
@@ -124,7 +124,7 @@ void CFG::sanityCheck(core::Context ctx) {
     }
 
     for (auto &bb : this->basicBlocks) {
-        ENFORCE(bb->bexit.isCondSet(), "Block exit condition left unset for block {}", bb->toString(ctx));
+        ENFORCE(bb->bexit.isCondSet(), "Block exit condition left unset for block {}", bb->toString(ctx.state));
```

Comment:
I don't think it's worth changing, but FWIW, instances of `Context` can automatically get coerced to `GlobalState`, so lines like these can stay the same.